include/seadsa/BottomUp.hh

```diff
@@ -32,20 +33,24 @@ private:
   const llvm::DataLayout &m_dl;
   llvm::TargetLibraryInfoWrapperPass &m_tliWrapper;
   const AllocWrapInfo &m_allocInfo;
+  const SpecGraphInfo &m_specGraphInfo;
```

Comment:
Can we come up with a more descriptive name than `SpecGraphInfo`? 

I do not like the combination of three very generic terms: *Specification*, *Grap*, and *Information*

Reply:
I agree, but I struggled coming up with a name, since at its core the pass just grabs all the spec functions and puts them in a map. It's the other passes that actually have to do the work of using the spec information provided. Do you have any suggestions?

Reply:
`DsaLibFuncInfo` -- dsa-specific information for known library functions